api/stt_test.go

```diff
@@ -0,0 +1,72 @@
+package api_test
+
+import (
+	"testing"
+	"encoding/binary"
+
+	"github.com/nkansal96/aurora-go/api"
+	"github.com/nkansal96/aurora-go/api/backend"
+	"github.com/nkansal96/aurora-go/audio"
+	"github.com/nkansal96/aurora-go/config"
+	"github.com/nkansal96/aurora-go/errors"
+	"github.com/stretchr/testify/require"
+)
+
+func createEmptyWAVFile() []byte {
```

Comment:
If this function was copied and pasted, then create a new package called `test` and put this function in there. Include that package wherever you need to use this function.

Reply:
Put the `test` package in the root of the SDK. i.e. `aurora-go/test`